environment-dev.yml

```diff
@@ -31,6 +31,8 @@ dependencies:
  - xarray
  - shapely
  - pydap>3.2.2|<3.2.2
+ - cxx-compiler
+ - sysroot_linux-64
```

Comment:
i guess the circle CI code has a separate compiler as it completed successfully before.

Reply:
Probably the reason is that we have `apt install g++` in the circle ci config file, so it would be using the system's globally installed c++ compiler, and not anything from conda.